org.abapgit.adt.ui/src/org/abapgit/adt/ui/internal/repositories/ModifiedObjectsForRepository.java

```diff
@@ -0,0 +1,33 @@
+package org.abapgit.adt.ui.internal.repositories;
```

Comment:
A new class created to maintain the modified objects for each repository. This also acts as the input type for the TreeViewer for selection of modified objects. These modified objects are the abapGit objects which can be overwrite objects or package warning objects.